build.gradle

```diff
@@ -143,17 +143,28 @@ repositories {
         //gigaherz maven for 1.16.2 mappings
         url 'https://www.dogforce-games.com/maven/'
     }
+    maven {
+        //JEI 1.16.3 tries to load 'mods:ftb-gui-library'
+        url 'https://maven.latmod.com/'
+    }
 }
 
 dependencies {
 //    minecraft "net.minecraftforge:forge:${version_minecraft}-${version_forge}"
     minecraft "net.minecraftforge:forge:1.16.3-34.0.7"
 
-    //runtimeOnly fg.deobf("mezz.jei:jei-1.16.2:7.1.0.11")
+    runtimeOnly fg.deobf("mezz.jei:jei-1.16.3:7.3.2.29")
     //runtimeOnly fg.deobf("nbtedit:NBTEdit:0.9.0")
-    compileOnly fg.deobf("mezz.jei:jei-1.16.2:7.3.2.25:api")
+    compileOnly fg.deobf("mezz.jei:jei-1.16.3:7.3.2.29:api")
     compile "malte0811:BlockModelSplitter:1.2.0"
+    //Apparently JEI 1.16.3 can't be applied here without...
+    compile fg.deobf('com.feed_the_beast.mods:ftb-gui-library:1.0.0.18')
```

Comment:
Why do we need a JEI update or a gui lib? It has worked so far without.

Reply:
In the build.gradle that is currently on that branch, JEI is only added as compile only dependency on a 1.16.2 version.
Since I'm pretty sure I'm supposed to check that the changes are in JEI, I added a 1.16.3 JEI dependency.
I first tried using the 1.16.2 version, but that gave me gradle issues, I'll try again, though.

If there's another way for me to add JEI to the runtime classpath that you prefer, I can use that instead.